README.md

```diff
@@ -83,7 +96,9 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+ES5
+not anonymous
+add()
```

Comment:
ES6. See [Method Properties](http://es6-features.org/#MethodProperties). Also, this has to be called as ```obj.add(1.2,77);```